docs/configuring.rst

```diff
@@ -192,6 +198,24 @@ Miscellaneous options
     :type: str
     :default: ``hidden``
 
+.. confval:: version_selector
+
+    Display a version selector below the title.
+    This feature makes usage of `Read the Docs Addons <https://docs.readthedocs.io/page/addons.html>`_ for this,
+    so it's required the documentation to be hosted on Read the Docs.
```

Comment:
I do wonder about adapting this so it can use a JSON file as well in a future version, but seems fine for now.

Reply:
```suggestion
    This feature makes usage of `Read the Docs Addons <https://docs.readthedocs.io/page/addons.html>`_ for this,
    so it's required the documentation to be hosted on Read the Docs.
    It only appears when there are more than 1 active version. 
```